spec/acceptance/cs_commit_spec.rb

```diff
@@ -102,7 +102,7 @@ class { 'corosync':
   end
 
   it 'creates the vip resource in the cib' do
-    command = if fact('default_provider') == 'pcs'
+    command = if fact('default_provider') == 'pcs' && fact('os.name') != 'Ubuntu'
```

Comment:
do you know why those changes are needed / how it worked in the past?

Reply:
Trying to figure it out right now. Default provider is set in spec_helper_acceptance for the tests, but somehow that does not work on ubuntu

Reply:
This should not happen IMHO:
```
Debug: Facter: fact "operatingsystem" has resolved to: Ubuntu
Warning: Found multiple default providers for cs_property: crm, pcs; using crm
Warning: Found multiple default providers for cs_primitive: crm, pcs; using crm
Warning: Found multiple default providers for cs_colocation: crm, pcs; using crm
Warning: Found multiple default providers for cs_commit: crm, pcs; using crm
```